gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/source/BackfillHiveSource.java

```diff
@@ -74,4 +74,15 @@ public boolean shouldCreateWorkunit(Partition sourcePartition, LongWatermark low
     // If no whitelist is set, all partitions of a dataset are backfilled
     return true;
   }
+
+  @Override
+  public boolean isOlderThanLookback(Partition partition) {
```

Comment:
Seems like a confusing behavior. Lookback can / should be applicable for whitelisted partition(s) as well.